tenable/io/v3/vm/agent_groups.py

```diff
@@ -0,0 +1,178 @@
+'''
+Agent Groups
+============
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`agent groups <agent-groups>` API endpoints.
+
+Methods available on ``tio.v3.vm.agent_groups``:
+
+.. rst-class:: hide-signature
+.. autoclass:: AgentGroupsAPI
+    :members:
+'''
+from typing import Dict, Union
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+
+
+class AgentGroupsAPI(ExploreBaseEndpoint):
+    _path: str = f'api/v3/agent-groups'
+    _conv_json: bool = True
+
+    def add_agent(self, group_id: int, *agent_ids: int) -> Union[None, Dict]:
+        '''
+        Adds an agent or multiple agents to the agent group specified.
+
+        :devportal:`agent-groups: add-agent <agent-groups-add-agent>`
+
+        Args:
+            group_id (int): The id of the group
+            agent_ids (int): The id of the agent
+
+        Returns:
+            :obj:`dict` or :obj:`None`:
+                If adding a singular agent, a :obj:`None` response will be
+                returned.  If adding multiple agents, a :obj:`dict` response
+                will be returned with a task record.
+
+        Examples:
+            Adding a singular agent:
+
+            >>> tio.v3.vm.agent_groups.add_agent(1, 1)
+
+            Adding multiple agents:
+
+            >>> tio.v3.vm.agent_groups.add_agent(1, 1, 2, 3)
+        '''
+
+        if len(agent_ids) <= 1:
+            # if there is only 1 agent id, we will perform a singular add.
+            self._put(f'{group_id}/agents/{agent_ids[0]}')
+        else:
+            # If there are many agent_ids, then we will want to perform a
+            # bulk operation.
+            return self._post(
+                f'{group_id}/agents/_bulk/add',
+                json={'items': [i for i in agent_ids]}
+            )
+
+    def configure(self, group_id: int, name: str) -> Dict:
+        '''
+        Renames an existing agent group.
+
+        :devportal:`agent-groups: configure <agent-groups-configure>`
+
+        Args:
+            group_id (int): The id of the group
+            name (str): The new name for the agent group
+
+        Returns:
+            :obj:`dict`
+
+        Examples:
+            >>> tio.v3.vm.agent_groups.configure(1, 'New Name')
+        '''
+        return self._put(f'{group_id}', json={'name': name})
+
+    def create(self, name: str) -> Dict:
+        '''
+        Creates a new agent group.
+
+        :devportal:`agent-groups: create <agent-groups-create>`
+
+        Args:
+            name (str): The name of the agent group
+
+        Returns:
+            :obj:`dict`:
+                The dictionary object representing the newly minted agent group
+
+        Examples:
+            >>> group = tio.v3.vm.agent_groups.create('New Agent Group')
+        '''
+        return self._post(f'', json={'name': name})
```

Comment:
You can simply write:
return self._post(json={'name': name})

Reply:
done.